examples/tokenvm/avalanche-ops.aws.md

```diff
@@ -0,0 +1,230 @@
+
+### Step 1
+
+Install [Rust](https://www.rust-lang.org/tools/install) to compile avalanche-ops, or download from the [latest release page](https://github.com/ava-labs/avalanche-ops/releases/tag/latest):
+
+```bash
+git clone git@github.com:ava-labs/avalanche-ops.git
+cd ./avalanche-ops
+./scripts/build.release.sh
+./target/release/avalancheup-aws --help
+```
+
+```bash
+# to download on Mac
+export LINUX_ARCH_TYPE=$(uname -m)
+echo ${LINUX_ARCH_TYPE}
+rm -f ./avalancheup-aws.${LINUX_ARCH_TYPE}-apple-darwin
+wget "https://github.com/ava-labs/avalanche-ops/releases/download/latest/avalancheup-aws.${LINUX_ARCH_TYPE}-apple-darwin"
+mv ./avalancheup-aws.${LINUX_ARCH_TYPE}-apple-darwin /tmp/avalancheup-aws
+chmod +x /tmp/avalancheup-aws
+/tmp/avalancheup-aws --help
+```
+
+```bash
+# to download on Linux
+export LINUX_ARCH_TYPE=$(uname -m)
+echo ${LINUX_ARCH_TYPE}
+rm -f ./avalancheup-aws.${LINUX_ARCH_TYPE}-linux-gnu
+wget "https://github.com/ava-labs/avalanche-ops/releases/download/latest/avalancheup-aws.${LINUX_ARCH_TYPE}-linux-gnu"
+mv ./avalancheup-aws.${LINUX_ARCH_TYPE}-linux-gnu /tmp/avalancheup-aws
+chmod +x /tmp/avalancheup-aws
+/tmp/avalancheup-aws --help
+```
+
+### Step 2
+
+Now we can spin up a new network of 6 nodes with some defaults:
+- `avalanche-ops` supports [Graviton-based processors](https://aws.amazon.com/ec2/graviton/) (ARM64). Use `--arch-type arm64` to run nodes in ARM64 CPUs.
+- `avalanche-ops` supports [EC2 Spot instances](https://aws.amazon.com/ec2/spot/) for cost savings. Use `--instance-mode=spot` to run instances in spot mode.
+
+```bash
+# launch/download all artifacts for x86 platform
+avalancheup-aws default-spec \
+--arch-type amd64 \
+--rust-os-type ubuntu20.04 \
+--anchor-nodes 3 \
+--non-anchor-nodes 3 \
+--region us-west-2 \
+--instance-mode=on-demand \
+--instance-size=2xlarge \
+--ip-mode=elastic \
+--metrics-fetch-interval-seconds 60 \
+--network-name custom \
+--keys-to-generate 5
+```
+
+The default commands above will download the public release binaries from github. To test your own binaries, use the following flags to upload to S3. These binaries must be built for the target remote machine platform (e.g., build for `aarch64` and Linux to run them in Graviton processors):
+
+```bash
+--upload-artifacts-aws-volume-provisioner-local-bin ${AWS_VOLUME_PROVISIONER_BIN_PATH} \
+--upload-artifacts-aws-ip-provisioner-local-bin ${AWS_IP_PROVISIONER_BIN_PATH} \
+--upload-artifacts-avalanche-telemetry-cloudwatch-local-bin ${AVALANCHE_TELEMETRY_CLOUDWATCH_BIN_PATH} \
+--upload-artifacts-avalanched-aws-local-bin ${AVALANCHED_AWS_BIN_PATH} \
+--upload-artifacts-avalanchego-local-bin ${AVALANCHEGO_BIN_PATH} \
+```
+
+It is recommended to specify your own artifacts to avoid flaky github release page downloads.
+
+### Step 3
+
+The `default-spec` command above will output the following `apply` and `delete` commands that you can copy and paste:
+
+```bash
+# run the following to create resources
```

Comment:
Specify that you need to populate an ENV before running this command to get access credentials.